.editorconfig

```diff
@@ -1,20 +1,371 @@
-# top-most EditorConfig file
+###############################################################################
```

Comment:
Configuration options from the previous are kept and the remaining defaults are explicitly listed. This ensures that any user-specific VS configuration doesn't take precedence instead.

Reply:
I wasn't aware this was still in the repo. Ideally we would use the one from Standards

https://github.com/SixLabors/Standards/blob/8b085c0ec4fb64797b9965741f7138f8f66a6b44/.editorconfig

Reply:
Super happy to update the Standards one with these defaults.

Reply:
This needs to live in the appropriate repo root either way, as it is hierarchical in nature and wouldn't be impactful from the standards directory.

Reply:
I think we can add it from anywhere as long as it's added to the solution and marked as root.

https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019#add-and-remove-editorconfig-files

Reply:
`editorconfig` isn't some standard that Microsoft/Visual Studio created, it is an external standard that we adopted because of its wide-spread use.

The way it is meant to be used (and the way it works) is described both here: https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019#file-hierarchy-and-precedence and at the source of truth here: https://editorconfig.org/#file-location.

Things won't function (in VS and in other editors) if it isn't located in the current directory or a parent of the current directory.

Reply:
Man that's such an incredibly frustrating design. I'm going o have to rethink the entire way I enforce standards now. (╯°□°）╯︵ ┻━┻